audit-service/README.md

```diff
@@ -18,8 +18,31 @@ limitations under the License.
 
 # Audit Service
 
-The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages contain all the details of the initial request and any other relevant information. This information will be passed to any local audit services that have
-been implemented. This service does not have any output Kafka topics.
+The Audit service is an implementation of the [AuditService Interface](src/main/java/uk/gov/gchq/palisade/service/audit/service/AuditService.java) which audits any incoming messages.
+
+The service accepts incoming messages on the `error` and `success` Kafka topics. These messages contain information about the original request and, depending on the message type, will contain either of the following:
+
+1. In the case of an error message, all the details of the error. 
+1. In the case of a success message, the id value of the returned resource.
+
+This information will be passed to any local audit services that have been implemented. This service does not have any output Kafka topics.
+
+## High level architecture
+
+<!--- 
+See audit-service/doc/audit-service.drawio for the source of this diagram
+--->
+![Audit Service diagram](doc/audit-service.png)
+
+## Flow of Control
```

Comment:
Can you colour co-ordinate the flow diagram, kafka topics in that purple/pink colour please!

Reply:
Updated colours